content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -837,6 +837,30 @@ checking if the Job has the annotation
 this annotation from Jobs. Instead, you can recreate the Jobs to ensure they
 are tracked using Pod finalizers.
 
+### Elastic Indexed Jobs
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+{{< note >}}
+In order to use this behavior, you must enable the `ElasticIndexedJob`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/).
+It is enabled by default.
+{{< /note >}}
+
+In v1.27 Elastic Indexed Jobs were introduced which allow scaling Indexed Jobs up/down
+by mutating both `.spec.Parallelism` and `.spec.Completions` together such that 
+`.spec.Paralellism` == `.spec.Completions`. Previously, the Batch API validation
+enforced the immutability of `.spec.Completions`, however, this validation was relaxed in
+1.27 to allow scaling Indexed Jobs up/down, subject to the constraints above.
+
+There are use cases where a batch workload requires an autoscaled indexed job with stable
+DNS pod names, for example MPI, Horovord, Ray, PyTorch etc. Prior to 1.27 this was not
```

Comment:
Remove everything starting "Prior to 1.27".

It's not explaining anything that wasn't mentioned already in the first paragraph.

Reply:
Docs are always explaining the current state. If you want to describe alternative approach I'd suggest creating a new section describing the behavior when you disable `ElasticIndexedJob`, although I doubt that's super useful. A short info about how this works is sufficient, like Aldo mentioned earlier.